content/influxdb/cloud-dedicated/admin/databases/list.md

```diff
@@ -15,31 +15,90 @@ related:
-2.  Run `influxctl database list` with the following:
+1.  If you haven't already, [download and install the `influxctl` CLI](/influxdb/cloud-dedicated/reference/cli/influxctl/#download-and-install-influxctl), and then [configure an `influxctl` connection profile](/influxdb/cloud-dedicated/reference/cli/influxctl/#configure-connection-profiles) for your cluster.
+2.  Run the `influxctl database list` command and provide the following:
 
-    - _(Optional)_ [Output format](#output-formats)
+    - _Optional_: [Output format](#output-formats)
 
 ```sh
 influxctl database list --format table
 ```
 
-### Output formats
+<!-------------------------------- END INFLUXCTL ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!------------------------------- BEGIN cURL ---------------------------------->
+_This example uses [cURL](https://curl.se/) to send a Management HTTP API request, but you can use any HTTP client._
+
+1. If you haven't already, follow the instructions to [install cURL](https://everything.curl.dev/install/index.html) for your system.
+2. In your terminal, use cURL to send a request to the following endpoint:
+
+   {{% api-endpoint endpoint="https://{{< influxdb/host >}}/api/v0/accounts/ACCOUNT_ID/clusters/CLUSTER_ID/databases" method="get" api-ref="/influxdb/cloud-dedicated/api/management/#operation/CreateClusterDatabase" %}}
+
+   In the URL, provide the following credentials:
+
+   - Your {{% product-name omit="Clustered" %}} cluster URL.
+   - `ACCOUNT_ID`: The ID of the [account](/influxdb/cloud-dedicated/get-started/setup/#request-an-influxdb-cloud-dedicated-cluster) that the cluster belongs to. To view account ID and cluster ID, [list cluster details](/influxdb/cloud-dedicated/admin/clusters/list/#detailed-output-in-json).
+   - `CLUSTER_ID`: The ID of the [cluster](/influxdb/cloud-dedicated/get-started/setup/#request-an-influxdb-cloud-dedicated-cluster) that you want to manage. To view account ID and cluster ID, [list cluster details](/influxdb/cloud-dedicated/admin/clusters/list/#detailed-output-in-json).
+
+   In request headers, provide the following:
+
+   - `Accept: application/json` to ensure the response body is JSON content
+   - `Content-Type: application/json` to indicate the request body is JSON content
+   - `Authorization: Bearer` and a [Management API token](/influxdb/cloud-dedicated/admin/tokens/management/) for your cluster _(see how to [create a management token](/influxdb/cloud-dedicated/admin/tokens/management/) for Management API requests)_.
+
+   The following example shows how to use the Management API to list databases in a cluster:
+
+   ```sh
+   curl \
+    --location "https://{{< influxdb/host >}}/api/v0/accounts/ACCOUNT_ID/clusters/CLUSTER_ID/databases" \
+    --header "Accept: application/json" \
+    --header "Authorization: Bearer MANAGEMENT_TOKEN" \
+    ```
```

Comment:
Any reason to not use code placeholders here? I know because it's nested in the ordered list, it will break, but where this is the last item in the list, I think you could probably un-nest this and add code placeholders.

Reply:
Nope. I must've forgot.